### DISCUSSION TITLE
How to deploy Ghostwriter in portainer?

### DISCUSSION THREAD

**Discussion by Richard-Simoni** [GhostManager/Ghostwriter#270] [Q&A]
Hello, thank you first of all for your software which is great, I use it daily for my work.
 I would like to deploy it for my company in a portainer environment linked to a gitlab.  Do you have any documentation about it? 
Thanks for your help,

**Comment by chrismaddalena** [comment]
Hey @Richard-Simoni,

I'm not familiar with Portainer at all. It says it manages Docker containers, so I see no reason it couldn't manage Ghostwriter's containers. The documentation for it is here:

https://docs.portainer.io/user/docker/containers/add

That says you just need the image information which you can pull from Docker CLI or the Docker Desktop app.
